vagrant/multi_node/hosts/localmesos-cluster

```diff
@@ -1,25 +1,25 @@
 #inventory file for local multi-node cluster environment
 
 [machines]
```

Comment:
Good call. This way /etc/hosts doesn't have to change when going between single and mutli